frontend/src/components/RoutePlanner/RoutePlanner.module.css

```diff
@@ -0,0 +1,93 @@
+ion-modal#favorite_dialogue {
+  --height: 25%;
+  --width: 90%;
+  --border-radius: 16px;
+  --box-shadow: 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1);
```

Comment:
Please don't use pixels as a unit. This makes the application unresponsive.

Reply:
Was adoped from existing route planner CSS. Will rework to use relative sizes